api/filter/http/ext_authz.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package envoy.api.v2.filter.http;
+
+import "google/protobuf/duration.proto";
+
+import "validate/validate.proto";
+
+// [#not-implemented-hide:]
+// External Authorization filter calls out to an external service over the 
+// gRPC Authorization API defined by api/external_auth.proto.
+// A failed check will cause this filter to return 403 Forbidden.
+message ExtAuthz {
+
+  // The name of the upstream authorization cluster.
+  string cluster_name = 1 [(validate.rules).string.min_bytes = 1];
+
+  // The timeout in milliseconds for the authorization service RPC. If not
```

Comment:
Need fail_policy   With default fail_open

Reply:
May I suggest allow|deny, rather than open|closed.

The latter is ambiguous, depending on whether you are thinking of a door, or a circuit (many software engineers are trained in EE).

Reply:
I totally agree. Lets use `allow|deny`

Reply:
Clarifying comment.
upstream may not be available or it is available but responds with “internal” or another processing error.
This policy applies in both cases.